go.mod

```diff
@@ -1,84 +1,76 @@
 module kubevirt.io/csi-driver
 
-go 1.19
+go 1.20
 
 require (
 	github.com/container-storage-interface/spec v1.6.0
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
-	github.com/golang/glog v1.0.0
 	github.com/golang/mock v1.6.0
-	github.com/golang/protobuf v1.5.2
+	github.com/golang/protobuf v1.5.3
 	github.com/google/uuid v1.3.0
+	github.com/gorilla/websocket v1.5.0
 	github.com/kubernetes-csi/csi-lib-utils v0.11.0
 	github.com/kubernetes-csi/csi-test/v5 v5.0.0
-	github.com/onsi/ginkgo/v2 v2.1.4
-	github.com/onsi/gomega v1.20.0
+	github.com/onsi/ginkgo/v2 v2.9.4
+	github.com/onsi/gomega v1.27.6
 	github.com/openshift/build-machinery-go v0.0.0-20200713135615-1f43d26dccc7
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.7.0
+	github.com/stretchr/testify v1.8.3
 	golang.org/x/net v0.19.0
-	google.golang.org/grpc v1.48.0
+	google.golang.org/grpc v1.56.3
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.24.2
-	k8s.io/apimachinery v0.24.2
+	k8s.io/api v0.28.6
+	k8s.io/apimachinery v0.28.6
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/klog/v2 v2.70.1
-	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
-	kubevirt.io/api v0.59.0
-	kubevirt.io/client-go v0.59.0
-	kubevirt.io/containerized-data-importer-api v1.56.0
+	k8s.io/klog/v2 v2.120.1
+	k8s.io/utils v0.0.0-20230505201702-9f6742963106
+	kubevirt.io/api v1.1.1
+	kubevirt.io/client-go v1.1.1
```

Comment:
we need client-go removed from the go.mod

Reply:
Removed